src/tracker.ts

```diff
@@ -370,15 +370,27 @@ class EditableTimestampField extends EditableField {
     }
 }
 
-function addEditableTableRow(tracker: Tracker, entry: Entry, table: HTMLTableElement, newSegmentNameBox: TextComponent, trackerRunning: boolean, file: string, getSectionInfo: () => MarkdownSectionInformation, settings: SimpleTimeTrackerSettings, indent: number): void {
+function addEditableTableRow(
+    tracker: Tracker,
+    entry: Entry,
+    table: HTMLTableElement,
+    newSegmentNameBox: TextComponent,
+    trackerRunning: boolean,
+    file: string,
+    getSectionInfo: () => MarkdownSectionInformation,
+    settings: SimpleTimeTrackerSettings,
+    indent: number
+): void {
```

Comment:
Is there a reason you changed this formatting? You didn't actually change any of the arguments, so it seems unnecessary

Reply:
I apologize, I have the default `Prettier` set for the dev program, I have already turned it off for this project, now I fix it